Chrome custom tabs - Prevent to display App Chooser Dialog

If multiple application that can handle the CustomTabIntent are installed on the device when trying to open the Chrome custom tabs the user is presented with the App Chooser Dialog.

Can we do anything to prevent this? Why would we  want the user to select something different?

Possible solution approach:
https://stackoverflow.com/questions/38429630/custom-chrome-tabs-asks-for-multiple-browser-to-choose

When I tried installing the new version. I am getting this error while building, any idea what could have gone wrong?

platforms/android/src/com/customtabplugin/ChromeCustomTabPlugin.java:133: error: method getPackageNameToUse in class CustomTabsHelper cannot be applied to given types;
        String packageName = CustomTabsHelper.getPackageNameToUse(cordova.getActivity(), url);
                                             ^
  required: Context
  found: Activity,String
  reason: actual and formal argument lists differ in length
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileArmv7DebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

--- a/src/android/ChromeCustomTabPlugin.java
+++ b/src/android/ChromeCustomTabPlugin.java
@@ -130,7 +130,7 @@ public class ChromeCustomTabPlugin extends CordovaPlugin{
 
         CustomTabsIntent customTabsIntent = builder.build();
         
-        String packageName = CustomTabsHelper.getPackageNameToUse(cordova.getActivity(), url);
+        String packageName = CustomTabsHelper.getPackageNameToUse(cordova.getActivity());
         if ( packageName != null ) {
            customTabsIntent.intent.setPackage(packageName);
         }

